src/libaktualizr/primary/aktualizr.h

```diff
@@ -215,6 +215,16 @@ class Aktualizr {
    */
   boost::signals2::connection SetSignalHandler(const SigHandler& handler);
 
+  /**
+   * Return a CA certificate, a client certificates a private key along with a Treehub URL,
+   * so an user (e.g. some Secondary) can setup a TLS connection with Treehub and download
+   * ostree repository revision objects.
+   * The aforementioned artifacts are returned in the form of a std::string containing gzip archive
+   *
+   * @return a string containing an archive with TLS certificates/credentials for a connection to Treehub
+   */
+  std::string GetTreehubTlsCreds() const { return uptane_client_->secondaryTreehubCredentials(); }
```

Comment:
It is a bit confusing why a generic primary's API call refers to "secondary" credentials internally. We assume that they are the same for primary and secondary, right?

Reply:
Yes, the method name is misleading, I am renaming it.